src/shared/kilocode/api.ts

```diff
@@ -29,4 +29,14 @@ export const kilocodeOpenrouterModels = {
 		outputPrice: 0.6,
 		isPromptCacheOptional: true,
 	},
+	"anthropic/claude-3.7-sonnet": {
+		maxTokens: 64_000,
+		contextWindow: 200_000,
+		supportsImages: true,
+		supportsPromptCache: true,
```

Comment:
Sanity check: does the caching implementation through openrouter work the same way (semantically) it did for anthropic?

Reply:
There is code for it.